libs/api/domains/license-service/src/lib/mappers/huntingLicenseMapper.ts

```diff
@@ -1,115 +1,147 @@
+            label: formatMessage(m.legalAddress),
             value: address,
           },
           {
             type: GenericLicenseDataFieldType.Value,
-            label: getLabel('cardNumber', locale, label),
+            label: formatMessage(m.cardNumber),
             value: t.number ?? '',
           },
           t.validFrom
             ? {
                 type: GenericLicenseDataFieldType.Value,
-                label: getLabel('publishedDate', locale, label),
+                label: formatMessage(m.publishedDate),
                 value: formatDate(t.validFrom),
               }
             : undefined,
           t.validFrom && t.validTo
             ? {
                 type: GenericLicenseDataFieldType.Value,
-                label: getLabel('validDuration', locale, label),
+                label: formatMessage(m.validDuration),
                 value: `${formatDate(t.validFrom)} - ${formatDate(t.validTo)}`,
               }
             : undefined,
           t.permitFor?.length
             ? {
                 type: GenericLicenseDataFieldType.Value,
-                label: getLabel('huntingPermitValidFor', locale, label),
+                label: formatMessage(m.huntingPermitValidFor),
                 value: t.permitFor?.map((p) => capitalize(p))?.join(', ') ?? '',
               }
             : undefined,
           t.benefits?.length
             ? {
                 type: GenericLicenseDataFieldType.Value,
-                label: getLabel('huntingPermitBenefits', locale, label),
+                label: formatMessage(m.huntingPermitBenefits),
                 value:
                   t.benefits?.map((b) => capitalize(b.land))?.join(', ') ?? '',
               }
             : undefined,
         ].filter(isDefined)
 
         return {
-          data,
-          rawData: JSON.stringify(t),
-          metadata: {
-            licenseNumber: t.number?.toString() ?? '',
-            licenseId: DEFAULT_LICENSE_ID,
-            expired: !t.isValid,
-            expireDate: t.validTo ? t.validTo.toISOString() : undefined,
-            links: [
-              {
-                label: getLabel('renewHuntingLicense', locale, label),
-                value:
-                  t.renewalUrl ??
-                  'https://innskraning.island.is/?id=gogn.ust.is',
-              },
-            ],
+          licenseName: formatMessage(m.huntingCard),
+          type: 'user',
+          payload: {
+            data,
+            rawData: JSON.stringify(t),
+            metadata: {
+              licenseNumber: t.number ?? '',
+              subtitle: formatMessage(m.licenseNumberVariant, {
+                arg: t.number ?? formatMessage(m.unknown),
+              }),
+              licenseId: DEFAULT_LICENSE_ID,
+              expired: !t.isValid,
+              expireDate: t.validTo ? t.validTo.toISOString() : undefined,
+              displayTag: expiryTag(
+                formatMessage,
+                t.isValid,
```

Comment:
expiry tag is expecting isExpired here

Reply:
`isExpired` is only for licenses which don't provide their own validity status. The hunting license provides this so it doesn't need any validity logic, if i'm not mistaken

Reply:
But could be related to this maybe?
> Veiðikort show expired on dev, but not expired on feature with same user 

👀

Reply:
maybe, i'll check it out